docs/standard/linq/creating-xml-trees.md

```diff
@@ -0,0 +1,425 @@
+---
+title: "Creating XML Trees - .NET"
+ms.date: 12/01/2019
+ms.assetid: cc74234a-0bac-4327-9c8c-5a2ead15b595
+---
+
+# Creating XML trees (C# and Visual Basic)
+This topic provides information about creating XML trees in C# and Visual Basic.
+
+## Creating XML trees in C&num;
+This section provides information about creating XML trees in C#.  
+  
+For information about using the results of LINQ queries as the content for an <xref:System.Xml.Linq.XElement>, see [Functional Construction (LINQ to XML)](./functional-construction-linq-to-xml.md).  
+  
+### Constructing elements
+The signatures of the <xref:System.Xml.Linq.XElement> and <xref:System.Xml.Linq.XAttribute> constructors let you pass the contents of the element or attribute as arguments to the constructor. Because one of the constructors takes a variable number of arguments, you can pass any number of child elements. Of course, each of those child elements can contain their own child elements. For any element, you can add any number of attributes.  
+  
+When adding <xref:System.Xml.Linq.XNode> (including <xref:System.Xml.Linq.XElement>) or <xref:System.Xml.Linq.XAttribute> objects, if the new content has no parent, the objects are simply attached to the XML tree. If the new content already is parented, and is part of another XML tree, the new content is cloned, and the newly cloned content is attached to the XML tree. The last example in this topic demonstrates this.  
+  
+To create a `contacts`<xref:System.Xml.Linq.XElement>, you could use the following code:  
+  
+```csharp  
+XElement contacts =  
+    new XElement("Contacts",  
+        new XElement("Contact",  
+            new XElement("Name", "Patrick Hines"),   
+            new XElement("Phone", "206-555-0144"),  
+            new XElement("Address",  
+                new XElement("Street1", "123 Main St"),  
+                new XElement("City", "Mercer Island"),  
+                new XElement("State", "WA"),  
+                new XElement("Postal", "68042")  
+            )  
+        )  
+    );  
+```  
+  
+If indented properly, the code to construct <xref:System.Xml.Linq.XElement> objects closely resembles the structure of the underlying XML.  
+  
+### XElement constructors  
+The <xref:System.Xml.Linq.XElement> class uses the following constructors for functional construction. Note that there are some other constructors for <xref:System.Xml.Linq.XElement>, but because they are not used for functional construction they are not listed here.  
```

Comment:
```suggestion

The <xref:System.Xml.Linq.XElement> class uses the following constructors for functional construction. Note that there are some other constructors for <xref:System.Xml.Linq.XElement>, but because they are not used for functional construction they are not listed here.  
```